warehouse/query-core/src/test/java/datawave/query/jexl/visitors/TreeFlatteningRebuildingVisitorTest.java

```diff
@@ -166,19 +169,42 @@ public void multipleNestingMixedOpsTest() throws Exception {
     }
     
     @Test
-    public void singleChildAndOrTest() throws Exception {
-        JexlNode eqNode = JexlASTHelper.parseJexlQuery("FIELD == 'value'");
+    public void singleChildAndOrTest() {
+        JexlNode eqNode = JexlNodeFactory.buildEQNode("FIELD", "value");
```

Comment:
This is a better test with this change.  Nice work.

Reply:
Thanks!